backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/agg/key/impl/ShipmentScheduleKeyValueHandler.java

```diff
@@ -81,6 +81,11 @@ public List<Object> getValues(@NonNull final I_M_ShipmentSchedule sched)
 		}
 
 		values.add(sched.getExternalHeaderId());
+		
+		if (sched.getExternalSystem_ID() > 0)
+		{
+			values.add(sched.getExternalSystem_ID());
```

Comment:
@dm-ps 

loud thinking: to be on the safe side, i think we shall do sometihng like
```java
values.add("EXTSYS"+sched.getExternalSystem_ID());
```

@mrie04  wdyt?

Reply:
@dm-ps @teosarca my proposal https://github.com/metasfresh/metasfresh/pull/21519